hare/algorithm.go

```diff
@@ -20,6 +20,7 @@ const protoName = "HARE_PROTOCOL"
 type Rolacle interface {
 	Eligible(layer types.LayerID, round int32, committeeSize int, id types.NodeId, sig []byte) (bool, error)
 	Proof(id types.NodeId, layer types.LayerID, round int32) ([]byte, error)
+	IsIdentityActive(edId string, layer types.LayerID) (bool, error)
```

Comment:
since we already have another `IsIdentityActive` (that queries atxdb) calling this one with the same name it is a bit confusing. Please rename to `IsIdentityActiveOnConsensusView`

Reply:
OnConsensusView is implementation is a bit impl detail but since it has all been moved to the oracle we have no plans to change it I'll rename it.